source/extensions/bootstrap/wasm/config.cc

```diff
@@ -14,19 +14,18 @@ namespace Extensions {
 namespace Bootstrap {
 namespace Wasm {
 
-static const std::string INLINE_STRING = "<inline>";
+void WasmServiceExtension::serverInitialized(Server::Configuration::ServerFactoryContext& context) {
+  createWasm(context);
```

Comment:
I don't think we should delay the whole bootstrap extension creation to server initialization. The VM can be still created early, and the bootstrap extension could get this event in later stage and it should make http call there.

Reply:
What is the advantage of splitting the VM initialization?
I'm asking as i think this will make the implementation significantly more complex - How would a wasm extension know when is it safe to perform an http call? that might need changes in the host sdk, and potentially in the ABI.